applications/asset_tracker_v2/src/main.c

```diff
@@ -534,7 +534,7 @@ void main(void)
 	int err;
 	struct app_msg_data msg;
 
-	if (!IS_ENABLED(CONFIG_LWM2M_CARRIER)) {
+	if (!IS_ENABLED(CONFIG_LWM2M_CARRIER) && !IS_ENABLED(CONFIG_NRF_CLOUD_FOTA)) {
```

Comment:
Just curious -- why was this change needed?

Reply:
without this change, `handle_nrf_modem_lib_init_ret` will read the modem lib init value and reboot before the nrf_cloud lib is able process the value.  this results in an unvalidated fota result on nrf cloud.

Reply:
Good catch.